miniapps/nurbs/nurbs_vector_diffusion.cpp

```diff
@@ -0,0 +1,396 @@
+// Simple test application that solves a diffusion using vector functions
+//
+// Compile with: make nurbs_vector_diffusion
+//
+// Sample runs:  nurbs_vector_diffusion
```

Comment:
The executables `nurbs_vector_diffusion` and `nurbs_stokes` should be included in `.gitignore`, as well as their output files.